util/semver/constraint.go

```diff
@@ -178,11 +182,13 @@ func (p *constraintParser) constraint() (*Constraint, error) {
 }
 
 /*
-	orList = span // See value method below.
-		| andList
-		| orList '||' andList // NPM, Default only.
-		| orList ',' andList // Maven and NuGet only.
-	span = VERSION ' ' '-' ' ' VERSION // NPM, Default only. Spaces required.
+orList = span // See value method below.
```

Comment:
This comment got changed for the worse, but I can't figure out how to make the new formatter accept it. Oh well.

Reply:
:crying_cat_face: